deploy/008-configure-acm-commands-aggregator.ts

```diff
@@ -0,0 +1,682 @@
+  ];
+};
+
+const getComptrollerPermissions = (): string[][] => {
+  return [
+    accounts.flatMap(timelock => [
+      ethers.constants.AddressZero,
+      "setCollateralFactor(address,uint256,uint256)",
+      timelock,
+    ]),
+    accounts.flatMap(timelock => [ethers.constants.AddressZero, "setMarketBorrowCaps(address[],uint256[])", timelock]),
+    accounts.flatMap(timelock => [
+      ethers.constants.AddressZero,
+      "setActionsPaused(address[],uint256[],bool)",
+      timelock,
+    ]),
+    accounts.flatMap(timelock => [ethers.constants.AddressZero, "setForcedLiquidation(address,bool)", timelock]),
+    accounts.flatMap(timelock => [ethers.constants.AddressZero, "unlistMarket(address)", timelock]),
+    [ethers.constants.AddressZero, "setCloseFactor(uint256)", AccountType.NORMAL_TIMELOCK],
+    [ethers.constants.AddressZero, "setLiquidationIncentive(uint256)", AccountType.NORMAL_TIMELOCK],
+    [ethers.constants.AddressZero, "setMinLiquidatableCollateral(uint256)", AccountType.NORMAL_TIMELOCK],
+  ];
+};
+
+const getVTokenPermissions = (): string[][] => {
+  return [
+    accounts.flatMap(timelock => [ethers.constants.AddressZero, "setReserveFactor(uint256)", timelock]),
+    accounts.flatMap(timelock => [ethers.constants.AddressZero, "setInterestRateModel(address)", timelock]),
+    accounts.flatMap(timelock => [ethers.constants.AddressZero, "setReduceReservesBlockDelta(uint256)", timelock]),
+    [ethers.constants.AddressZero, "setProtocolSeizeShare(uint256)", AccountType.NORMAL_TIMELOCK],
+  ];
+};
+
+const getRewardDistributorPermissionsTimebased = (): string[][] => {
+  return [
+    [ethers.constants.AddressZero, "setRewardTokenSpeeds(address[],uint256[],uint256[])", AccountType.NORMAL_TIMELOCK],
+    [
+      ethers.constants.AddressZero,
+      "setLastRewardingBlockTimestamps(address[],uint256[],uint256[])",
+      AccountType.NORMAL_TIMELOCK,
+    ],
+  ];
+};
+
+const getRewardDistributorPermissionsBlockbased = (): string[][] => {
+  return [
+    [ethers.constants.AddressZero, "setRewardTokenSpeeds(address[],uint256[],uint256[])", AccountType.NORMAL_TIMELOCK],
+    [ethers.constants.AddressZero, "setLastRewardingBlocks(address[],uint32[],uint32[])", AccountType.NORMAL_TIMELOCK],
+  ];
+};
+
+const getIRMPermissions = (): string[][] => {
+  return [
+    [ethers.constants.AddressZero, "updateJumpRateModel(uint256,uint256,uint256,uint256)", AccountType.NORMAL_TIMELOCK],
+  ];
+};
+
+const getConverterPermissions = (): string[][] => {
+  return [
+    accounts.flatMap(timelock => [ethers.constants.AddressZero, "pauseConversion()", timelock]),
+    accounts.flatMap(timelock => [ethers.constants.AddressZero, "resumeConversion()", timelock]),
+    accounts.flatMap(timelock => [ethers.constants.AddressZero, "setMinAmountToConvert(uint256)", timelock]),
+    [
+      ethers.constants.AddressZero,
+      "setConversionConfig(address,address,ConversionConfig)",
+      AccountType.NORMAL_TIMELOCK,
+    ],
+  ];
+};
+
```

Comment:
I miss a function to authorize timelocks to execute functions on the `XVSVaultTreasury` contract. Every timelock contract should be authorized to execute `fundXVSVault(uint256)` on that contract